README.md

```diff
@@ -1,32 +1,34 @@
-# Examples of the Cloud Dataflow SDK for Java 
+# Examples of the Cloud Dataflow SDK for Java and Python
 
 [Google Cloud Dataflow](https://cloud.google.com/dataflow/) provides a simple,
 powerful programming model for building both batch and streaming parallel data
 processing pipelines. This repository hosts example pipelines that use the [Cloud
-Dataflow SDK for Java](https://cloud.google.com/dataflow/model/programming-model) and demonstrate the basic functionality of Google Cloud Dataflow. 
+Dataflow SDK](https://cloud.google.com/dataflow/model/programming-model) and demonstrate the basic functionality of Google Cloud Dataflow. 
 
 ## Word Count
 
 A good starting point for new users is our set of
-[word count](https://github.com/GoogleCloudPlatform/DataflowJavaSDK-examples/blob/master/src/main/java/com/google/cloud/dataflow/examples) examples, which computes word frequencies.  This series of four successively more detailed pipelines is described in detail in the accompanying [walkthrough](https://cloud.google.com/dataflow/examples/wordcount-example).
+word count ([java](java/examples/src/main/java/com/google/cloud/dataflow/examples), [python](python/dataflow_examples)) examples, which computes word frequencies.  This series of four successively more detailed pipelines is described in detail in the accompanying [walkthrough](https://cloud.google.com/dataflow/examples/wordcount-example).
```

Comment:
walkthrough link should go to Beam, not Dataflow, because Dataflow one is 1.x only